ops/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+gaescaffoldserver:
+  image: rehabstudio/appengine-dev
+  command: make -C /app run
+  user: root
+  working_dir: /app
+  volumes:
+    - ../src/server:/app
+    - /home/aeuser
+    - ../ops:/home/aeuser/creds
+  ports:
+    - 0.0.0.0:8080:8080
+    - 8000:8000
+
+gaescaffoldclient:
+  image: node:5.0.0
```

Comment:
5.4.1 is the latest stable release, what's the thoughts on using just node:5 or node:5.4 instead of 5.0.0?
See https://hub.docker.com/_/node/ for reference